metrics/core/src/test/java/tech/pegasys/pantheon/metrics/prometheus/PrometheusMetricsSystemTest.java

```diff
@@ -14,32 +14,32 @@
 
 import static java.util.Arrays.asList;
 import static java.util.Collections.emptyList;
+import static java.util.Collections.singleton;
 import static java.util.Collections.singletonList;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static tech.pegasys.pantheon.metrics.MetricCategory.JVM;
-import static tech.pegasys.pantheon.metrics.MetricCategory.NETWORK;
-import static tech.pegasys.pantheon.metrics.MetricCategory.PEERS;
-import static tech.pegasys.pantheon.metrics.MetricCategory.RPC;
+import static tech.pegasys.pantheon.metrics.PantheonMetricCategory.JVM;
+import static tech.pegasys.pantheon.metrics.PantheonMetricCategory.NETWORK;
+import static tech.pegasys.pantheon.metrics.PantheonMetricCategory.PEERS;
+import static tech.pegasys.pantheon.metrics.PantheonMetricCategory.RPC;
 
 import tech.pegasys.pantheon.metrics.Counter;
 import tech.pegasys.pantheon.metrics.LabelledMetric;
-import tech.pegasys.pantheon.metrics.MetricCategory;
 import tech.pegasys.pantheon.metrics.MetricsSystem;
 import tech.pegasys.pantheon.metrics.Observation;
 import tech.pegasys.pantheon.metrics.OperationTimer;
 import tech.pegasys.pantheon.metrics.OperationTimer.TimingContext;
+import tech.pegasys.pantheon.metrics.PantheonMetricCategory;
 import tech.pegasys.pantheon.metrics.noop.NoOpMetricsSystem;
 
 import java.util.Comparator;
-import java.util.EnumSet;
 
 import org.junit.Test;
 
 public class PrometheusMetricsSystemTest {
 
   private static final Comparator<Observation> IGNORE_VALUES =
-      Comparator.comparing(Observation::getCategory)
+      Comparator.<Observation, String>comparing(o -> o.getCategory().getName())
```

Comment:
Category is no longer an enum so the comparator has to be based off the name instead of the enum ordinal (which is probably a good thing anyway). Not sure why the generics is being difficult but it really wants to be told explicitly.